mmtk/Cargo.toml

```diff
@@ -22,6 +22,7 @@ openjdk_version = "ca90b43f0f51d9ddf754e6ab134c5030cf54118b"
 libc = "0.2"
 lazy_static = "1.1"
 once_cell = "1.10.0"
+spin = "0.9.2"
```

Comment:
If we're not using `spin` (it seems like we're not) then we should remove it.

Reply:
Thanks! Removed